backend/tracim_backend/command/cleanup.py

```diff
@@ -247,7 +247,7 @@ def _delete_user_database_info(
         anonymize_if_required: bool = False,
         anonymized_user_display_name: typing.Optional[str] = None,
     ):
-        print('trying to delete user {}: "{}"\n'.format(user.user_id, user.email))
+        print('trying to delete user {}: "{}"\n'.format(user.user_id, user.email or user.username))
```

Comment:
Email first here ? Why if username is first in other part of the code ?

Reply:
> so we decide to show username first here ?
> and here we decide to show email first ?
> username first here ? ok.
> Email first here ? Why if username is first in other part of the code ?

Hmm, simply because I didn't notice

Reply:
To avoid those confusions, maybe define a `login` property on `User` that returns `self.email or self.username` ?

Reply:
good idea, i do that

Reply:
ok.